src/pages/generate.tsx

```diff
@@ -87,10 +75,36 @@ export default function ClientIdentifierGenerator() {
     defaultMaxAge: undefined,
   };
 
+  /** Helper to load form state and values from storage or defaults. */
+  const getSessionStorageStateOrDefault: () => {
```

Comment:
it'd be an idea to add a `max-age` to this, such that coming back after a day doesn't restore data, it's really only for "I changed pages and oops I actually wanted that data"

Reply:
Addressed in https://github.com/inrupt/client-identifier-helper/pull/185/commits/ec0e24242114d647bbf77652364bc7891793ae7e